WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -162,6 +164,12 @@ extension ReaderTabView {
         containerView.translatesAutoresizingMaskIntoConstraints = false
         childController.view.translatesAutoresizingMaskIntoConstraints = false
 
+        controller.children.forEach {
+            $0.willMove(toParent: nil)
```

Comment:
According to the docs, you shouldn't have to do this since `removeFromParent` is called below:

> The removeFromParent() method automatically calls the didMove(toParent:) method of the child view controller after it removes the child.

https://developer.apple.com/documentation/uikit/uiviewcontroller/1621405-didmove